build/debian-base/Makefile

```diff
@@ -51,6 +51,10 @@ ifeq ($(ARCH),s390x)
 	BASEIMAGE?=s390x/debian:stretch
 	QEMUARCH=s390x
 endif
+ifeq ($(ARCH),mips64le)
+        BASEIMAGE?=inspurcloud/debian-debootstrap:mips64el-stretch
```

Comment:
I'm not sure k8s should be shipping a third party image like this

cc @cblecker @spiffxp @dims

Reply:
agree with @BenTheElder 
 
/hold

Reply:
Multiarch is an image repository provided by qemu.
multiarch/debian-debootstrap:mips64el-stretch was built by qemu using debootstrap. This is its production script.
It's a mistake that I use the inspurcloud.

Reply:
I see https://hub.docker.com/u/mips64le org but don't see anything published under this repo? all of those images mentioned here for other architectures built and well maintained by the docker-library project,